.github/workflows/milestone.yml

```
@@ -8,6 +8,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: master
```

I'm not 100% sure, but this might cover the pull request to different branches issue:

```suggestion
          ref:  ${{ github.event.pull_request.base.ref }}
```

---

You're right, the backport case slipped my mind, ill double check this in a test repo